### PR TITLE
Added tests to check autofstrim on volume on destination (DR)

### PR DESCRIPTION
**Pull request by gejain**
Added tests to check autofstrim on volume on destination (DR)

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PTX-17074

**Special notes for your reviewer**:
Test Run Result - https://aetos.pwx.purestorage.com/resultSet/testSetID/172774/testCaseID/886476

### CODE REVIEW

**Review by Rohit-PX — APPROVED** [review]

File: tests/testTriggers.go [hunk 1]
```diff
@@ -6691,6 +6693,130 @@ func TriggerConfluentAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *E
...
+					// Set cluster context to cluster where torpedo is running
+					ScheduleValidateClusterPair(appContexts[0], false, true, defaultClusterPairDir, false)
+				})
+			}
+		}
+	})
+
+	log.Infof("Migration Namespaces: %v", migrationNamespaces)
+
+	time.Sleep(2 * time.Minute)
```
Comment (lsrinivas-pure) [inline tests/testTriggers.go:6778]: any particular reason for this wait?

File: tests/testTriggers.go [hunk 2]
```diff
@@ -6691,6 +6693,130 @@ func TriggerConfluentAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *E
...
+	log.Info("Start migration")
```
Comment (lsrinivas-pure) [inline tests/testTriggers.go:6779]: use infof

File: tests/testTriggers.go (hunk 1 above)
Comment (gejain) [reply tests/testTriggers.go]: It was added as a part of testing the code, but I forgot to remove. Its done. Thanks!.
New test run -
https://aetos.pwx.purestorage.com/resultSet/testSetID/173527/testCaseID/890963

File: tests/testTriggers.go (hunk 2 above)
Comment (gejain) [reply tests/testTriggers.go]: Done.

**Review by lsrinivas-pure — APPROVED** [review]
lgtm

**Pull request merged**